### PR TITLE
Add some debug prints at startup

### LINKED ISSUE #105: Add some debug prints to debug why WBD has a startup delay
As per https://github.com/robotology/icub-tech-support/issues/999#issuecomment-706312574

### PR DESCRIPTION
**Pull request by S-Dafarra**
Fixes #105

### CODE REVIEW

File: devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp [hunk 1]
```diff
@@ -1318,6 +1318,9 @@ bool WholeBodyDynamicsDevice::open(os::Searchable& config)
 {
     std::lock_guard<std::mutex> guard(this->deviceMutex);
 
+    double tick = yarp::os::Time::now();
+    yDebug() << "wholeBodyDynamics Statistics: Opening estimator.";
```
Comment (prashanthr05) [inline devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp:1322]: Isn't it better to call tick after the debug print? Print is time-expensive, maybe?

Comment (prashanthr05) [reply devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp:1322]: - Print 
- tick
- tock
- Print

Comment (S-Dafarra) [reply devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp]: Done in https://github.com/robotology/whole-body-estimators/pull/106/commits/65fabfd81f7fc25a246d80a61bd3e0dc0ae0df20

**Review by prashanthr05 — APPROVED** [review]
Thanks! Could you just add a line in the Changelog?

**Comment by S-Dafarra** [comment]
> Thanks! Could you just add a line in the Changelog?

Sure, done in https://github.com/robotology/whole-body-estimators/pull/106/commits/5a6030d2a7db297b058f845d77e899ae436049b0

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,10 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+- Added some debug prints to detect where the wholeBodyDynamics device hangs at startup. (See [!106](https://github.com/robotology/whole-body-estimators/pull/106)).
 
 ## [0.4.0] - 2021-02-11
 ### Added
-- - Add `HW_USE_MAS_IMU` feature in `wholeBodyDynamics` device to enable the use YARP's Multiple Analog Sensor interface based IMU measurements. (See [!99](https://github.com/robotology/whole-body-estimators/pull/99))
+- Add `HW_USE_MAS_IMU` feature in `wholeBodyDynamics` device to enable the use YARP's Multiple Analog Sensor interface based IMU measurements. (See [!99](https://github.com/robotology/whole-body-estimators/pull/99))
```
Comment (prashanthr05) [inline CHANGELOG.md:12]: Thanks for this fix ;D

**Pull request merged**